database/neo4j/src/main/java/com/neueda/jetbrains/plugin/graphdb/database/neo4j/bolt/Neo4jBoltDatabase.java

```diff
@@ -35,13 +35,23 @@ public Neo4jBoltDatabase(Neo4jBoltConfiguration configuration) {
 
     @Override
     public GraphQueryResult execute(String query) {
+        return execute(query, null);
```

Comment:
Let's not pass null's when not need.
We can use `Collections.emptyMap()`.

In this way we do not need `if` statement below.